### PR TITLE
Documentation website

### PR DESCRIPTION
**Pull request by Gsantomaggio**

### CODE REVIEW

**Comment by lukebakken** [comment]
This is cool! Is the idea to publish this to GitHub Pages?

**Comment by Gsantomaggio** [comment]
@lukebakken yep!

**Comment by lukebakken** [comment]
Here is how we do it in the AMQP .NET client - https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/main/APIDOCS.md

I created an "orphaned" `gh-pages` branch to start:

```
git checkout --orphan gh-pages
```

...then added that as a submodule to the `main` branch. Seems to work well.

**Comment by Gsantomaggio** [comment]
Already did it ! Thank you!
The wip: https://rabbitmq.github.io/rabbitmq-stream-dotnet-client/stable/htmlsingle/index.html

File: docs/asciidoc/super-streams.adoc [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+Use `rabbitmq-streams add_super_stream --help` to learn more about the command.
+
+[[super-stream-producer]]
+===== Publishing to a Super Stream
+
+When the topology of a super stream like the one described above has been set, creating a producer for it is straightforward:
+
+.Creating a Producer for a Super Stream
+[source,java,indent=0]
```
Comment (Zerpet) [inline docs/asciidoc/super-streams.adoc:78]: No Java! 👮‍♂️

File: docs/asciidoc/super-streams.adoc [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+This hash function provides good uniformity and it is compatible with the other clients.
+
+
+====== Deduplication
+
+Deduplication for a super stream producer works the same way as with a <<api.adoc#outbound-message-deduplication, single stream producer>>.
+The publishing ID values are spread across the streams but this does affect the mechanism.
+
+.Creating a DeduplicatingProducer for a Super Stream
+[source,java,indent=0]
```
Comment (Zerpet) [inline docs/asciidoc/super-streams.adoc:102]: 👮

File: docs/asciidoc/super-streams.adoc [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+The next subsections cover the basic settings of a super stream consumer and a <<super-stream-sac, dedicated section>> covers how super stream consumers and single active consumer play together.
+
+====== Declaring a Super Stream Consumer
+
+Declaring a super stream consumer is not much different from declaring a single stream consumer.
+The `Consumer.Create(..)` must be used to set the super stream to consume from:
+
+.Declaring a super stream consumer
+[source,java,indent=0]
```
Comment (Zerpet) [inline docs/asciidoc/super-streams.adoc:141]: 🚓

File: docs/asciidoc/super-streams.adoc [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+
+The example above uses by default <<api.adoc#consumer-manual-offset-tracking, manual offset tracking>>.
+It looks up the latest stored offset when a consumer becomes active to start consuming at the appropriate offset and it stores the last dispatched offset when a consumer becomes inactive.
+
+The story is not the same with <<api.adoc#consumer-manual-offset-tracking, manual offset tracking>> as the client library does not know which offset it should store when a consumer becomes inactive.
+The application developer can use the <<api.adoc#consumer-update-listener, `ConsumerUpdateListener)` callback>> to react appropriately when a consumer changes state.
+The following snippet illustrates the use of the `ConsumerUpdateListener` callback:
+
+.Using manual offset tracking for a super stream single active consumer
+[source,java,indent=0]
```
Comment (Zerpet) [inline docs/asciidoc/super-streams.adoc:272]: 🚔

**Pull request merged**